buildconfig/stubs/pygame/math.pyi

```diff
@@ -18,91 +18,195 @@ if sys.version_info >= (3, 9):
 else:
     from typing import Collection
 
-_VectorTypeVar = TypeVar("_VectorTypeVar", Vector2, Vector3)
+_TVec = TypeVar("_TVec", bound=_GenericVector)
+
+# not implemented in code, only implemented here for ease of implementing
+# typestubs. Contains attributes/methods common to Vector2 and Vector3
+# Also used with _TVec generics
+class _GenericVector(Collection[float]):
+    epsilon: float
```

Comment:
Good old epsilon, typed, implemented, but not documented.

Reply:
does anyone know what it's for, lol? Might be worth document it if it has some significance

Reply:
https://en.wikipedia.org/wiki/Machine_epsilon

Reply:
This already got documented in #3373